docs/QUICKSTART.md

```diff
@@ -0,0 +1,46 @@
+# Quick Start for KeePassXC
```

Comment:
Maybe submit this document also to the website repository, so people can view it online? This file won't be packaged with binary releases.

Reply:
That is a great idea. 

I wonder if any of the current team would like to update the KeePassXC.org site: I haven't worked on a Jekyll site in a while, and also don't fully understand the expectations for its organization. Thanks.